src/main/java/com/icthh/xm/ms/balance/config/AsyncConfiguration.java

```diff
@@ -35,7 +36,7 @@ public AsyncConfiguration(JHipsterProperties jhipsterProperties) {
 
     @Override
     @Bean(name = "taskExecutor")
-    public Executor getAsyncExecutor() {
+    public TaskExecutor getAsyncExecutor() {
```

Comment:
why this change is made?

Reply:
taskExecutor bean is injected into: `com.icthh.xm.ms.balance.config.DatabaseConfiguration#liquibase` line 111, param 1